go/vt/callinfo/plugin_grpc.go

```diff
@@ -19,11 +19,11 @@ package callinfo
 // This file implements the CallInfo interface for gRPC contexts.
 
 import (
-	"fmt"
-	"html/template"
-
 	"context"
+	"fmt"
```

Comment:
did you run `goimports`? i think this order is wrong

Reply:
Yeah, I think this order is right?

Reply:
yup, my eyes blurred i guess, sorry about that!